web/index.html

```diff
@@ -468,6 +468,29 @@ <h5 class="modal-title" id="donateModalLabel">Donation Links</h5>
 			</div>
 		</template>
 
+			<!-- NSFW Warning Modal -->
+			<div class="modal fade" id="disclaimerModal" tabindex="-1" aria-labelledby="disclaimerModalLabel" aria-hidden="true">
+				<div class="modal-dialog modal-dialog-centered">
+					<div class="modal-content">
+						<div class="modal-header">
+							<h5 class="modal-title" id="disclaimerModalLabel">Place Atlas disclaimer</h5>
```

Comment:
Either I would have "Atlas disclamer" or just "Disclaimer". I prefer just "Disclaimer", but if you prefer the long one, make sure to use "r/place Atlas Disclaimer" (note the "r/place)

Reply:
Done